### ISSUE TITLE
In your Makefile, `stf_ios_mirrorfeed` is missing

### ISSUE THREAD

**Issue by jay0129** [DeviceFarmer/stf_ios_support#67]
The repo of `stf_ios_mirrorfeed` is missing. So I had to remove it.
Is it okay without `stf_ios_mirrorfeed` in `repos` directory?

**Comment by tepmehatop** [comment]
same problem, `fix` add tou tour makefile
`repos/stf_ios_mirrorfeed:
	git clone https://github.com/tmobile/stf_ios_mirrorfeed.git repos/stf_ios_mirrorfeed
`

**Comment by jay0129** [comment]
@tepmehatop , it doesn't work.
```
·> make clone
make: *** No rule to make target `//github.com/tmobile/stf_ios_mirrorfeed.git', needed by `clone'.  Stop.
```

My Makefile:
```
# --- REPO CLONES ---

clone: repos/stf_ios_mirrorfeed repos/WebDriverAgent repos/osx_ios_device_trigger repos/stf-ios-provider

repos/stf_ios_mirrorfeed:
        git clone https://github.com/tmobile/stf_ios_mirrorfeed.git repos/stf_ios_mirrorfeed

repos/stf-ios-provider/package.json: repos/stf-ios-provider

repos/stf-ios-provider:
        $(eval REPO=$(shell jq '.repo_stf // "https://github.com/nanoscopic/stf-ios-provider.git"' config.json -j))
        git clone $(REPO) repos/stf-ios-provider --branch master

repos/ios_video_stream:
        git clone https://github.com/nanoscopic/ios_video_stream.git repos/ios_video_stream
```

**Comment by gkrananth** [comment]
I got the latest repo and run make command getting the following error.

fatal: not a git repository (or any of the parent directories): .git
parse error: Invalid numeric literal at line 1, column 3
parse error: Invalid numeric literal at line 1, column 3
parse error: Invalid numeric literal at line 1, column 3
git clone  repos/WebDriverAgent --branch 
error: option `branch' requires a value
make: *** [repos/WebDriverAgent] Error 129

**Comment by nanoscopic** [comment]
I updated the check version script to avoid the "parse error... invalid..." errors. That should not occur any more.

The clone command should just have the bit for stf_ios_mirrorfeed removed.
The intent was to remove it for now until that method of video streaming is stabilized, but I failed to remove all references to it.
It is safe to just remove the remaining references to it and I will do so also and update.
